superset-frontend/src/features/alerts/components/NotificationMethod.tsx

```diff
@@ -76,6 +76,9 @@ export const NotificationMethod: FunctionComponent<NotificationMethodProps> = ({
   }
 
   const onMethodChange = (method: NotificationMethodOption) => {
```

Comment:
Ideally the `onMethodChange` shouldn't fire if the value isn't changed. It's likely worthwhile checking that the `Select` component logic is correct.

Reply:
Opening a new issue https://github.com/apache/superset/issues/27668